src/openeo_odc/map_processes_odc.py

```diff
@@ -137,7 +153,10 @@ def convert_from_node_parameter(args_in, from_par=None):
                 and isinstance(item, dict) \
                 and 'from_parameter' in item \
                 and item['from_parameter'] in from_par.keys():
-            args_in[k] = '_' + from_par[item['from_parameter']]
+            if isinstance(from_par[item['from_parameter']], str):
+                args_in[k] = '_' + from_par[item['from_parameter']]
+            else:
+                args_in[k] = from_par[item['from_parameter']]
```

Comment:
I removed this if statement, because I changed `oeop.array_element(**{'data': [1, 1, 1], 'index': 0})` to `oeop.array_element(**{'data': _parameters, 'index': 0})` but I think I need to put it back in, to resolve the `from_parameter` correctly. :)

Reply:
Yes then we have our example :wink: